### PR TITLE
CCV-540: Android: Coach Toolbox - "Email Group" button URL change.

### PR DESCRIPTION
**Pull request by santosmodernclimate**
* GroupDetailFragment.cs: Added logic for the message group button. The button opens a 
   webview to the specified URL.

* FragmentGroupDetail.axml: Changed the layout to accomodate the email
  group button.

### CODE REVIEW

File: Droid/Fragments/GroupDetailFragment.cs [hunk 1]
```diff
@@ -21,6 +21,7 @@ public class GroupDetailFragment : Android.Support.V4.App.Fragment
         public const string TAG = "group-detail-fragment";
         private const string GROUP = "group";
         private const string LABEL = "label";
+        private const string EMAIL_GROUP_URL = "https://ccv.church/page/1751?group=";
```
Comment (jhawkzz) [inline Droid/Fragments/GroupDetailFragment.cs]: I believe this URL is in our shared code config. If not, could we move it there and have iOS use it as well?

Comment (dwee9) [reply Droid/Fragments/GroupDetailFragment.cs]: Yes it is in shared `COACH_EMAIL_ALL_URL`

Comment (santosmodernclimate) [reply Droid/Fragments/GroupDetailFragment.cs]: Changed to use the Strings file and simplified the code a bit

File: Droid/Fragments/GroupDetailFragment.cs [hunk 2]
```diff
@@ -407,6 +414,12 @@ public override View OnCreateView( LayoutInflater inflater, ViewGroup container,
             return view;
         }
 
+        private void EmailGroupClick( )
+        {
+            Uri builder = Uri.Parse( URLS.COACH_EMAIL_ALL_URL ).BuildUpon( ).AppendQueryParameter( "group", group.Id.ToString( ) ).Build( );
+            StartActivity( ActivityIntentFactory.FromUrl( Context, builder.ToString( ) ).CreateIntent( true, false ) );
```
Comment (dwee9) [inline Droid/Fragments/GroupDetailFragment.cs]: The Query Parameter names are also in that strings file `QUERY_NAMES`

**Pull request merged**